### ISSUE TITLE
Update the accessibility severity rubric

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6502]
Reconsider the Level 0 and Level 1 items, and consider adding:

* Incomplete e2e tests ( no axe checks, not enough axe checks, etc )
* Plans for keyboard-e2e tests as best practices
* Best practices section

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md

**Comment by shiragoodman** [comment]
@1Copenut has this been completed or are we still waiting to make these updates?

**Comment by 1Copenut** [comment]
@shiragoodman These are still to be done. I needed to get the 508 guidance and a few other things done before getting this work moving.

**Comment by shiragoodman** [comment]
@1Copenut is this still relevant?

**Comment by 1Copenut** [comment]
@shiragoodman Yes. I think it got lost in the shuffle of other things we've had going on, but these updates are still valid.

**Comment by shiragoodman** [comment]
ok im adding the TTT label
